templates/CRM/Sumfields/Form/SumFields.Custom.tpl

```diff
@@ -0,0 +1,30 @@
+{literal}
+<script type="text/javascript">
+  CRM.$(window).load(function() {
+    if (CRM.$('#show_simplified').prop("checked") === true){
+      switch_simplified();
+    } else {
+      switch_normal();
+    }
+    CRM.$('#show_simplified').change(function(){
+      if(this.checked) {
+        switch_simplified();
+      } else {
+        switch_normal();
+      }
+    });
+  });
+  function switch_simplified() {
+      CRM.$('.CRM_Sumfields_Form_SumFields tr.crm-sumfields-form-block-sumfields_active_fundraising_fields td.value input[id$="simplified"]').show();
+      CRM.$('.CRM_Sumfields_Form_SumFields tr.crm-sumfields-form-block-sumfields_active_fundraising_fields td.value label[for$="simplified"]').show();
+      CRM.$('.CRM_Sumfields_Form_SumFields tr.crm-sumfields-form-block-sumfields_active_fundraising_fields td.value input').not('[id$="simplified"]').prop( "checked", false ).hide();
+      CRM.$('.CRM_Sumfields_Form_SumFields tr.crm-sumfields-form-block-sumfields_active_fundraising_fields td.value label').not('[for$="simplified"]').hide();
+  }
+  function switch_normal() {
+      CRM.$('.CRM_Sumfields_Form_SumFields tr.crm-sumfields-form-block-sumfields_active_fundraising_fields td.value input[id$="simplified"]').prop( "checked", false ).hide();
+      CRM.$('.CRM_Sumfields_Form_SumFields tr.crm-sumfields-form-block-sumfields_active_fundraising_fields td.value label[for$="simplified"]').hide();
+      CRM.$('.CRM_Sumfields_Form_SumFields tr.crm-sumfields-form-block-sumfields_active_fundraising_fields td.value input').not('[id$="simplified"]').show();
+      CRM.$('.CRM_Sumfields_Form_SumFields tr.crm-sumfields-form-block-sumfields_active_fundraising_fields td.value label').not('[for$="simplified"]').show();
+  }
+</script>
+{/literal}
```

Comment:
Based on your comment on the ticket... perhaps we should slightly change the logic to be: 

hide simplified (default)
show simplified

In other words, if you show the simplified you see both simplified and non-simplified together. It does make it a bit more overwhelming, but I think you are right - unchecking and making existing fields suddenly disappear might not be a great user experience. Also, I suppose some people might want to mix and match.

Reply:
> In the ticket you also expressed concern about people needing to re-build smart groups. That's always been the case when you change fields, so I don't think we need to solve it in order to get this change merged. Also, if people are looking for the simplified fields, then it is probably because their existing smart groups are broken ;).

Initially, my concern was mainly on existing working installations and shifting them from one source to the other without breaking the smartgroups. But i can't see an easy fix on that and if it's too complex, it's easier to just recreate the groups per installation. :)